pkg/embedded/templates/m2/settings.xml

```diff
@@ -0,0 +1,28 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+     <pluginGroups>
+         <pluginGroup>dev.galasa</pluginGroup>
+     </pluginGroups>
+     
+     <profiles>
+         <profile>
+             <id>galasa</id>
+             <activation>
+                 <activeByDefault>true</activeByDefault>
+             </activation>
+             <repositories>
+                 <repository>
+                     <id>galasa.repo</id>
+                     <url>https://development.galasa.dev/main/maven-repo/obr</url>
+                 </repository>
+             </repositories>
+             <pluginRepositories>
+                 <pluginRepository>
+                     <id>galasa.repo</id>
+                     <url>https://development.galasa.dev/main/maven-repo/obr</url>
```

Comment:
need to refer to the official repo, not the development one.

I guess you could add some comment into the XML file to say 'if you want to use the bleeding-edge version of things, use this repository instead:...'